repos/spack_repo/builtin/packages/py_onnxruntime/package.py

```diff
@@ -191,6 +219,15 @@ def setup_run_environment(self, env: EnvironmentModifications) -> None:
         value = self.dynamic_cpu_arch_values.index(value)
         env.set("MLAS_DYNAMIC_CPU_ARCH", str(value))
 
+    @property
+    def python_lib_dir(self):
+        python_vers_phrase = "python{0}".format(self.spec["python"].version.up_to(2))
+        return join_path("lib", python_vers_phrase)
+
+    @property
+    def site_packages_dir(self):
+        return join_path(self.python_lib_dir, "site-packages")
+
```

Comment:
Doesn't `python_platlib` do the same? (I can't seem to find where it is defined to know for sure, but quite a few packages use this to the same effect)